### ISSUE TITLE
Support localized ROMs via jumpers

### ISSUE THREAD

**Issue by apoloval** [apoloval/artemisa#99]
The ROM memories used by Artemisa, the SST39SF, support a minimum of 128KB of memory in its basic configuration (SST39SF010). That's 4x the size of a MSX System ROM. Thus, we can place ROMs localised for 4 different countries. Even more, if we decide to use other variants of the flash ROM:

- For SST39SF010, 128KB, 4 different ROM images.
- For SST39SF020, 256KB, 8 different ROM images.
- For SST39SF040, 512KB, 16 different ROM images.

In order to facilitate the production of equipments, it would be really useful to have some jumpers in the motherboard to select the 32KB segment of the ROM chip that will be used. So regardless the user localisation, we can distribute the same ROM image, and let the user to select the MSX System locale with jumpers.

**Comment by apoloval** [comment]
In MSX System, we have 11 different localisations.
```
;       CNTRY   Country
;               0 = Japan
;               1 = USA
;               2 = International
;               3 = UK
;               4 = France
;               5 = Germany
;               6 = Italy
;               7 = Spain
;               8 = Arabic
;               9 = Korea
;               10 = Russia
```

It means the optimal combination is using the SST39SF040, 512KB, 16 different ROM images. That's 4 jumpers. 

Another discussion is whether using 2 single pin rows with pull-up (or pull-down) resistors, or 3 pin rows with no resistors.

**Comment by apoloval** [comment]
Finally addressed in 1da3f15acf2bef044cd3811df884700acce66193. 

What we did is to add 4 jumpers, so we can choose from a set of 16 different MSX System ROMs of 32KB. The jumpers have 3 pins, and a pull-down resistor network just in case the user forgets to connect it.
